www-src/app/replicator/replicator_calendars.coffee

```diff
@@ -45,10 +49,13 @@ module.exports =
                         @initEventsInPhone body.last_seq, cb
             (cb) => @updateCalendars cb
             (cb) => @syncEventsPhone2Pouch cb
-            (cb) => @syncEventsToCozy cb
             (cb) => @syncEventsFromCozy cb
+            (cb) => async.eachSeries @changesFromCozy, @handleConflict.bind(@), cb
```

Comment:
@jacquarg I think it should be cleaner to use a fat arrow for declaring `handleConflict`, instead of `bind(@)`